pkg/glib

```diff
@@ -33,6 +33,7 @@ done
 
 sed -i 's;-Werror=;-W;g' configure
 
+case "$A" in i?86) optcflags="${optcflags} -march=${A}";; esac
```

Comment:
please explain was the build error was, this doesn't look like the optimal solution